### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by wimulkeman**
Updated and extended the documentation:
- Removed duplicate line. The resources.yml is allready included as a rexourse in the config file included above this line.
- Described how you can override the default price calculator which can be used to alter the variant price to the tier price in the overviews and detail pages.

### CODE REVIEW

**Comment by mamazu** [comment]
This is a very nice solution you documented. I have never worked with decorators in Sylius but this looks very easy to implement indeed.
Thanks for the merge request.

File: README.md [hunk 1]
```diff
@@ -45,6 +44,17 @@ bin/console translation:update
 * The Bundle overrides the `ProductVariant` class that is provided by Sylius. This will be overridden in the `resource.yml` of the Bundle. If you want to override that class in your application too, you have to merge the two configurations.
 * This bundle registers an [order processor](https://docs.sylius.com/en/1.2/components_and_bundles/components/Order/processors.html) service `brille24_tier_price.order_processing.order_prices_recalculator`. If you wish to use your own order processor or change its priority, you could register a [compiler pass](https://symfony.com/doc/current/service_container/compiler_passes.html).
 
+If you want to override the default price shown for a variant in the product overview and detail page, override the default product variant price calculation class of Sylius.
+```yaml
+  sylius.calculator.product_variant_price.decorator:
```
Comment (teohhanhui) [inline README.md]: It shouldn't be named like this... :)

**Comment by teohhanhui** [comment]
Should this be the default behaviour? Which is the more intuitive?

**Comment by wimulkeman** [comment]
The main problem is that by overriding the service by default from within the plugin, you prevent other people from extending the default service them selfs (they still can but it requires more digging through the code).

Aldo for my project it would have been the aspected behavior I like the fact that you can choose how to implement it for now.

File: README.md (hunk 1 above)
Comment (wimulkeman) [reply README.md]: Fixed by @mamazu in https://github.com/Brille24/SyliusTierpricePlugin/commit/c22bd8786b4b62d17d7f88ac4f7651eb5335e262

Comment (teohhanhui) [reply README.md]: That's the wrong fix, as we're declaring the service in the app, not in this bundle.

**Comment by teohhanhui** [comment]
> The main problem is that by overriding the service by default from within the plugin, you prevent other people from extending the default service them selfs (they still can but it requires more digging through the code).

That's not true. A service could be decorated many times. It should have no negative effect.

The real question should be: is this behaviour something that should happen out of the box? We should follow the principle of least surprise...

**Comment by wimulkeman** [comment]
@teohhanhui That is true, and my comment is indeed a bit confusing. What I meant is that by overriding it be default, people who want to override the Sylius default price calculator will override the TierPrice price calculator if I understand the Symfony docs correctly.

But because of the nature of this plugin, I would say that decorating the sylius price calculator by default will be expected as default, and you could provide textual coverage of it in the README.

**Pull request merged**